### PR TITLE
Add state to idea

### PR DESCRIPTION
**Pull request by alexliahushau**
Add state to idea

### CODE REVIEW

File: src/main/resources/db/sql/schema.sql [hunk 1]
```diff
@@ -20,6 +20,7 @@ CREATE TABLE `IDEA` (
   , `LATITUDE` DECIMAL(9,6) NULL
   , `LONGITUDE` DECIMAL(9,6) NULL
   , `IMAGE_URL` VARCHAR(512) NULL
+  , `STATE` VARCHAR(10) NULL
```
Comment (Hanna-Aliakseichykava) [inline src/main/resources/db/sql/schema.sql]: Please, fix also schema in mysql db script

**Comment by Hanna-Aliakseichykava** [comment]
:+1:

**Comment by SviatlanaHlukhava** [comment]
:+1:

**Comment by Hanna-Aliakseichykava** [comment]
:+1:

**Pull request closed without merging**